exit
__errno_location
